app/src/main/java/kth/jjve/xfran/models/Result.java

```diff
@@ -2,60 +2,53 @@
 
 import java.time.LocalDate;
 
-/**
+/*
  * Result constructor
  */
 
-
 public class Result {
 
     private Workout workout;
-    private String resultType;
-    private double time;
-    private double reps;
-    private double weight;
+    private String scoreType, comments, score; //Todo: deal better with different types of scores
     private Integer feelScore;
-    private String comments;
     private LocalDate date;
+    private boolean scaled, empty;
 
-    public Result(Workout workout, LocalDate date){
+    public Result(Workout workout, String score, Integer feelScore, String comments, LocalDate date, boolean scaled) {
         this.workout = workout;
+        this.score = score;
+        this.feelScore = feelScore;
+        this.comments = comments;
         this.date = date;
+        this.scaled = scaled;
+        this.empty = false;
```

Comment:
What does this do?

Reply:
Yes @jitseve that was the case, but then I don't allow it on the UI so it is not used. I forgot to delete it. Changed on new commit.